views/artist-search-results.hbs

```diff
@@ -0,0 +1,9 @@
+
+        <img src="{{imageResult.0.url}}" alt="artist photo">
+        
+        {{#each artistsResult}}
+            {{name}}
+        {{/each}}
+
+        <a href="/albums/someArtistIdGoesHere">View Album</a>
+

```

Comment:
Here you could loop on the `artistResult` to display all the artists (name, image, albums, etc)